src/components/access_group/AccessGroupView.vue

```diff
@@ -57,6 +63,20 @@ export default {
   props: {
     groupname: String,
   },
+  beforeRouteEnter(to, from, next) {
+    const { groupname } = to.params;
+    store.dispatch('setLoading');
+    const [membersPromises, membersResolvers] = fetchMembers(store, groupname);
+    const [agPromises, agResolvers] = fetchAccessGroup(store, groupname);
+    resolvePromisesSerially(
+      [...membersPromises, ...agPromises],
+      [...membersResolvers, ...agResolvers],
+    ).then(() => {
+      store.dispatch('completeLoading');
+      console.log('Complete accessgroup component route loading');
```

Comment:
some debugging leftover. Is there a way to just strip this for production builds? That would be cool. Of have log levels.